### PR TITLE
 bug fix to parsePDB 

### PR DESCRIPTION
**Pull request by jamesmkrieger**
Without the fix it says it parsed 2 PDB files when it parses atomgroups and headers.

### CODE REVIEW

File: prody/proteins/pdbfile.py [hunk 1]
```diff
@@ -138,8 +138,13 @@ def parsePDB(*pdb, **kwargs):
             results = results[0]
         results = list(results)
 
+        if len(results) == 2:
+            numPdbs = len(results[0])
+        else:
+            numPdbs = len(results)
+
```
Comment (SHZ66) [inline prody/proteins/pdbfile.py]: This seems to be a little arbitrary: the user may have two pdbs without headers. In that case `len(results[0])` would probably obtain the number of atoms in the first structure (?). Could you determine it from the `header` argument or some other variables?

Comment (jamesmkrieger) [reply prody/proteins/pdbfile.py]: I guess.

Comment (jamesmkrieger) [reply prody/proteins/pdbfile.py]: How about `if isinstance(results[0][0], Atomic) and isinstance(results[1][0], dict):`

Comment (jamesmkrieger) [reply prody/proteins/pdbfile.py]: ok that doesn't work if they are all headers

**Pull request merged**